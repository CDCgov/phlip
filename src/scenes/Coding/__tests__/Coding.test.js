import React from 'react'
import { shallow } from 'enzyme'
import { Coding } from '../index'

const props = {
  actions: {
    getNextQuestion: () => {},
    getPrevQuestion: () => {},
    getCodingOutlineRequest: () => {},
    answerQuestionRequest: () => {},
    onChangeComment: () => {},
    onChangePincite: () => {},
    onChangeCategory: () => {},
    onClearAnswer: () => {},
    onCloseCodeScreen: () => {}
  },
  question: {},
  projectName: 'Test Project',
  projectId: 1,
  outline: {},
  currentIndex: 0,
  questionOrder: [],
  categories: [],
  selectedCategory: 0,
  jurisdictionId: 1,
  userAnswers: {}
}

describe('Coding scene', () => {
  test('should render correctly', () => {
    expect(shallow(<Coding {...props} />)).toMatchSnapshot()
  })
})
