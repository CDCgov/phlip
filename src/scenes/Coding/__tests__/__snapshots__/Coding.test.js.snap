// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Coding scene should render correctly 1`] = `
<Container
  column={false}
  flex={true}
  spacing={0}
  style={
    Object {
      "display": "flex",
      "flexWrap": "nowrap",
      "height": "100%",
      "position": "relative",
      "width": "100%",
    }
  }
>
  <WithStyles
    currentQuestion={Object {}}
    open={false}
    selectedCategory={0}
  />
  <withRouter(Connect(HeaderedLayout))
    className=""
    padding={false}
  >
    <Column
      component={
        <WithStyles
          item={true}
        />
      }
      displayFlex={true}
      flex={true}
    >
      <withRouter(Header)
        empty={true}
        onJurisdictionChange={[Function]}
        projectId={1}
        projectName="Test Project"
        selectedJurisdiction={1}
      />
      <Container
        column={false}
        flex={true}
        spacing={0}
        style={
          Object {
            "backgroundColor": "#f5f5f5",
          }
        }
      >
        <Row
          component={
            <WithStyles
              item={true}
            />
          }
          displayFlex={true}
          flex={true}
          style={
            Object {
              "overflow": "auto",
            }
          }
        >
          <Column
            component={
              <WithStyles
                item={true}
              />
            }
            displayFlex={false}
            flex={false}
          />
          <Column
            component={
              <WithStyles
                item={true}
              />
            }
            displayFlex={true}
            flex={true}
            style={
              Object {
                "overflow": "auto",
                "padding": "1px 27px 10px 27px",
              }
            }
          />
        </Row>
      </Container>
    </Column>
  </withRouter(Connect(HeaderedLayout))>
</Container>
`;
